rtil/src/threads/ue/rebo/rebo_init.rs

```diff
@@ -1455,4 +1461,28 @@ fn get_camera_mode() -> u8 {
 #[rebo::function("Tas::set_camera_mode")]
 fn set_camera_mode(mode: u8) {
     AMyCharacter::set_camera_mode(mode);
-}
\ No newline at end of file
+}
+#[rebo::function("Tas::timer_start")]
+fn timer_start() {
+    livesplit::Timer::start();
+}
+#[rebo::function("Tas::timer_split")]
+fn timer_split() {
+    livesplit::Timer::split();
+}
+#[rebo::function("Tas::timer_reset")]
+fn timer_reset(update_splits: bool) {
+    livesplit::Timer::reset(update_splits);
+}
+#[rebo::function("Tas::get_game_time")]
+fn get_game_time() -> f64 {
+    livesplit::Timer::get_game_time()
+}
+#[rebo::function("Tas::set_game_time")]
+fn set_game_time(time: f64) {
+    livesplit::Timer::set_game_time(time);
+}
+#[rebo::function("Tas::pause_game_time")]
+fn pause_game_time() {
+    livesplit::Timer::pause_game_time();
+}
```

Comment:
Shouldn't these also start with `timer_*` considering they affect the timer-part?